### PR TITLE
MRG: Revision of Project 3: Dynamics of AS patterns

### PR DESCRIPTION
**Pull request by lynnzhao92**
1. Add revise folder, with images subfolder inside
2. Add revise_dynamics.md
3. Problem:
 - cannot make inline latex work in line 57, suggestion? @peterasujan 

Please Review. @peterasujan @shamindras @dylandaniels @jarrodmillman

### CODE REVIEW

**Comment by jarrodmillman** [comment]
You should be revising the text directly.   You also need to remove `.DS_Store` files.  That is junk the Mac OS X makes.  I added it to the `.gitignore` file.  How did you manage to commit it?  (I thought adding it to `.gitignore` would have prevented you from adding it, which is why I am asking.)

When you make the changes, you should over write your previous commit.

**Comment by peterasujan** [comment]
@lynnzhao it seems that Github's markdown renderer does not currently support LaTeX (see [here](https://github.com/github/markup/issues/274)). However, Sphinx (the documentation builder) should be able to handle this, so you should definitely keep it in there (it just won't look super pretty on Github).

**Comment by peterasujan** [comment]
@jarrodmillman I think the `.DS_Store` files got in there because she committed them before merging your updated `.gitignore`. And once something is versioned I think you can keep committing it, despite it being in the `.gitignore`.

**Comment by lynnzhao92** [comment]
@jarrodmillman @peterasujan Do you think our current revision is OK and just modifying the dynamics.md file instead of adding a new file? Or you want us mainly answer the questions of dynamics.md? 
I will figure out the .DS_STORE thing. Thx!

File: doc/revise/revise_dynamics.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
+# Revision of Project 3: Dynamics of AS Patterns
```
Comment (jcyhong) [inline doc/revise/revise_dynamics.md]: Remove "Revision of ".

File: doc/revise/revise_dynamics.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+```
+
+
+## Testing Framework Outline
+* Main idea: check the reasonability of IST to be the number we choosed and the AS probability or Transition probability by testing: 
+* Examples: 
+	+ For each mice, The AS numbers for Night Time bins < The  AS numbers of Daytime Bins.
+	+ P(IS|Food) < P(AS|Food) 
+
+## Further Questions:
```
Comment (jcyhong) [inline doc/revise/revise_dynamics.md]: Rename this section as "Additional Remarks" for consistency.

File: doc/revise/revise_dynamics.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
+# Revision of Project 3: Dynamics of AS Patterns
+
+## Statement of Problem
+The objective of Dynamics Analysis is to classify the active states and inactive states.
+
```
Comment (jcyhong) [inline doc/revise/revise_dynamics.md]: In addition to classifying the AS and the IS, you may also want to mention that one of the key objectives is to understand the dynamics/transitions between AS and IS.

**Comment by lynnzhao92** [comment]
Cannot figure out the .DS_Store

**Comment by jarrodmillman** [comment]
Please rebase instead of merging in the future.

**Pull request merged**